templates/test/mocha.opts.tmpl

```diff
@@ -3,4 +3,5 @@
 --require tsconfig-paths/register
 --require source-map-support/register
 --require ./test/_setup.ts
+--file ./test/unit/_global_hooks.ts
```

Comment:
No need to add this here, and it will break other tests

Reply:
@diego-G pointed out that a test breaks when run individually with `npx mocha` due to sandbox not being cleaned properly. so in case someone wants to run the test individually like `TS_NODE_PROJECT=./test/tsconfig.json npx nyc npx mocha test/unit/peer_pool.ts` there is a setup missing somewhere. running `npm run test` normally works fine. I think it is a bit of edge case but to satisfy running the test specifically via `TS_NODE_PROJECT=./test/tsconfig.json npx nyc npx mocha test/unit/peer_pool.ts` I will add an extra `sandbox.restore` in the `beforeEach` of the file. If you see this extra restore in the code this is the reason why. @diego-G

Reply:
but most of the library doesn't have this structure, so it will not pass the other tests.
Also, _global_hook is automatically caught by the mocha runtime

Reply:
I've added a commit to solve the issue. @shuse2 @mitsujutsu